### PR TITLE
Add stats page

### PR DESCRIPTION
**Pull request by Ewoutvans**
Feedback welcome!

Here is a screenshot:
screenshot-2018-1-20 stats

### CODE REVIEW

**Comment by ryantheleach** [comment]
@Ewoutvans Looking good, is there an associated issue?

**Comment by progwml6** [comment]
this is part of #329

File: app/views/users/admin/stats.scala.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                            label: "Unsafe Downloads",
+                            backgroundColor: "lightseagreen",
+                            borderColor: "darkseagreen",
+                            fill: false,
+                            data: @Html(Json.stringify(Json.toJson(unsafeDownloads.map(_._1).toList)))
+                        }]
+                    },
+                    options: {
+                        title: {
+                            text: "Reviews"
```
Comment (phase) [inline app/views/users/admin/stats.scala.html]: This should be `Downloads`

File: app/controllers/Application.scala [hunk 2]
```diff
@@ -186,4 +188,29 @@ final class Application @Inject()(data: DataHelper,
     Redirect(ShowHome)
   }
 
+  /**
+    * Show stats
+    * @return
+    */
+  def showStats() = (Authenticated andThen PermissionAction[AuthRequest](ViewStats)) { implicit request =>
+    val queryReviews: Future[Seq[(String,String)]] = this.service.DB.db.run(sql"SELECT foo.data, CAST(foo.when AS DATE) FROM (WITH backup AS (SELECT 0,current_date - (INTERVAL '1 day' * generate_series(0, 9)) days) SELECT COALESCE(data.count, 0) AS data, COALESCE(data.when, backup.days) AS when FROM (SELECT COUNT(*), CAST(approved_at AS DATE) AS when FROM project_versions WHERE approved_at IS NOT NULL GROUP BY CAST(approved_at AS DATE)) data FULL OUTER JOIN backup ON data.when = backup.days) AS foo".as[(String, String)])
```
Comment (phase) [inline app/controllers/Application.scala]: This looks ugly and unmaintainable. You could make one template SQL file and modify it for the different queries.

**Review by phase — CHANGES_REQUESTED** [review]
This looks good except for the SQL queries.

File: app/controllers/Application.scala (hunk 2 above)
Comment (Ewoutvans) [reply app/controllers/Application.scala]: maybe we should make it a sql view that would be more suitable in this situation
what do you think is the best way?

Comment (simon816) [reply app/controllers/Application.scala]: at the very least, break up the query into multiple lines

File: app/views/users/admin/stats.scala.html (hunk 1 above)
Comment (Ewoutvans) [reply app/views/users/admin/stats.scala.html]: This is fixed now but you placed your comment on the wrong line :)

File: app/controllers/Application.scala (hunk 2 above)
Comment (phase) [reply app/controllers/Application.scala]: A twirl template might be overcomplicating things. A file with some placeholders would do fine.

**Review by kashike — APPROVED** [review]

**Review by phase — APPROVED** [review]

**Comment by progwml6** [comment]
This PR has conflicts that need to be resolved

**Pull request merged**